docs/en/ecosystem_release/starrocks-kubernetes-operator.md

```diff
@@ -40,9 +40,62 @@ For example, the download URL for kube-starrocks chart v1.8.6 is:
 
 ## Release notes
 
-## 1.8
+### 1.9
 
-### 1.8.6
+#### 1.9.1
+
+**Improvements**
+
+- **[Helm Chart]** The operator will not create PersistentVolumeClaim (PVC) for log storage when you set `logStorageSize` to `0`.  [#398](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/398)
```

Comment:
when you set `logStorageSize` to `0` -> when/if when `logStorageSize` is set to `0`